### PR TITLE
fix: onboarding tip header casing

### PR DESCRIPTION
**Pull request by m4thieulavoie**
Resolves https://github.com/brave/brave-browser/issues/17120

## Submitter Checklist:

- [x] I confirm that no security/privacy review [is needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or that I have [requested](https://github.com/brave/security/issues/new/choose) one
- [x] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [x] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [x] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [x] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [x] Ran `git rebase master` (if needed)

## Reviewer Checklist:

- [ ] A security review [is not needed](https://github.com/brave/brave-browser/wiki/Security-reviews), or a link to one is included in the PR description
- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

**Review by bsclifton — APPROVED** [review]
LGTM! Thank you 😄

**Comment by bsclifton** [comment]
Adding @emerick in case exact casing affects Greaselion

File: components/resources/strings/brave_components_resources_en-GB.xtb [hunk 1]
```diff
@@ -376,7 +376,7 @@
 <translation id="2070991379950719711">Start using Brave Rewards</translation>
 <translation id="6318696841371191107">Take a quick tour</translation>
 <translation id="8465790416279415448">By proceeding, you agree to the <ph name="TOS_LINK_BEGIN"/>Terms of Service<ph name="TOS_LINK_END"/> and <ph name="PP_LINK_BEGIN"/>Privacy Policy<ph name="PP_LINK_END"/>.</translation>
-<translation id="2404165674840145940">Tip with Brave rewards</translation>
+<translation id="2404165674840145940">Tip with Brave Rewards</translation>
```
Comment (emerick) [inline components/resources/strings/brave_components_resources_en-GB.xtb:379]: No need to modify this file; it's auto-generated and then uploaded to our translators.

**Comment by emerick** [comment]
This won't affect Greaselion (thanks for asking @bsclifton!) Looks good, just pointed out that there's no need to modify any `.XTB` files AFAIK.

File: components/resources/strings/brave_components_resources_en-GB.xtb (hunk 1 above)
Comment (m4thieulavoie) [reply components/resources/strings/brave_components_resources_en-GB.xtb:379]: Should I revert that change?

Comment (emerick) [reply components/resources/strings/brave_components_resources_en-GB.xtb:379]: Yeah, I think that makes sense. The translators will update all of the .XTB's in one go the next time we order translations.

Comment (m4thieulavoie) [inline components/resources/strings/brave_components_resources_en-GB.xtb:379]:
```suggestion
<translation id="2404165674840145940">Tip with Brave rewards</translation>
```

File: components/resources/strings/brave_components_resources_en-GB.xtb [hunk 2]
```diff
@@ -865,4 +865,4 @@
 <translation id="6973157709854838218">Hide your balances from display</translation>
 <translation id="4548946945059463969">Reveal your balances on screen</translation>
 <translation id="242959166945779492">You do not have any balances in your account. Visit FTX to purchase some.</translation>
-</translationbundle>
\ No newline at end of file
+</translationbundle>
```
Comment (m4thieulavoie) [inline components/resources/strings/brave_components_resources_en-GB.xtb:868]:
```suggestion
</translationbundle>

```

File: components/resources/strings/brave_components_resources_en-GB.xtb (hunk 1 above)
Comment (m4thieulavoie) [reply components/resources/strings/brave_components_resources_en-GB.xtb]: Consider it done :D

**Review by emerick — APPROVED** [review]
LGTM

**Comment by bsclifton** [comment]
@m4thieulavoie can you squash this into one commit? You can basically run a `git rebase -i upstream/master` and pick squash for the 2nd/3rd commit 😄

**Comment by m4thieulavoie** [comment]
> @m4thieulavoie can you squash this into one commit? You can basically run a `git rebase -i upstream/master` and pick squash for the 2nd/3rd commit 😄

Yup, done

**Pull request merged**